components/frontend/src/index.js

```diff
@@ -1,9 +1,15 @@
 import "fomantic-ui-css/semantic.min.css"
 import "react-grid-layout/css/styles.css"
-import "react-resizable/css/styles.css"
 
 import { createRoot } from "react-dom/client"
 
+const error = console.error
+console.error = (...args) => {
+    if (/Support for defaultProps will be removed from function components/.test(args[0])) return
```

Comment:
Perhaps good to create tickets and reference them here in a comment, so we don't forget that these are suppressed?

Reply:
https://github.com/ICTU/quality-time/issues/8974